src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -0,0 +1,2718 @@
+    "mmorpg.stat.all_lightning_damage": "Урон молнией",
+    "mmorpg.stat.all_mana_leech": "§7Поглощение §a[VAL1]§7% вашего §d☀ Всего§7 урона как маны",
+    "mmorpg.stat.all_penetration": "Пробивание всех видов защиты",
+    "mmorpg.stat.all_physical_damage": "Физический урон",
+    "mmorpg.stat.all_resist": "Вся защита",
+    "mmorpg.stat.all_vuln_crit": "§7Весь урон всегда критует вас.",
+    "mmorpg.stat.all_water_damage": "Холодный урон",
+    "mmorpg.stat.area_cdr": "Уменьшение перезарядки заклинаний области",
+    "mmorpg.stat.area_dmg": "Урон по области",
+    "mmorpg.stat.area_spell_dmg": "Урон по области",
+    "mmorpg.stat.armor": "Броня",
+    "mmorpg.stat.armor_penetration": "Пробивание брони",
+    "mmorpg.stat.armor_per_10_dodge": "§a[VAL1] §9❈ Брони§7 за 10 §2❈ Очков уклонения§7",
+    "mmorpg.stat.armor_per_10_mana": "§a[VAL1] §9❈ Брони§7 за 10 §9☯ Очков маны§7",
+    "mmorpg.stat.arrow_cdr": "Уменьшение перезарядки стрелковых заклинаний",
+    "mmorpg.stat.arrow_spell_dmg": "Урон от стрел",
+    "mmorpg.stat.attack_atk_style_dmg": "Урон от атаки",
+    "mmorpg.stat.attack_damage": "Урон от атаки",
+    "mmorpg.stat.aura_effect": "Эффект ауры",
+    "mmorpg.stat.axe_damage": "Урон от топора",
+    "mmorpg.stat.beast_cdr": "Уменьшение перезарядки заклинаний зверей",
+    "mmorpg.stat.beast_spell_dmg": "Урон от зверей",
+    "mmorpg.stat.bleed_chance": "Шанс на кровотечение",
+    "mmorpg.stat.bleed_damage": "Урон от кровотечения",
+    "mmorpg.stat.bleed_duration": "Продолжительность кровотечения",
+    "mmorpg.stat.bleed_proc_chance": "Шанс на раскалывание",
+    "mmorpg.stat.bleed_resistance": "Сопротивление кровотечению",
+    "mmorpg.stat.bleed_strength": "Сила кровотечения",
+    "mmorpg.stat.block_chance": "Шанс блока",
+    "mmorpg.stat.block_leech_cap": "Максимальная скорость поглощения блока",
+    "mmorpg.stat.blood": "Кровь",
+	"mmorpg.stat.blood_leech_cap": "Максимальная скорость поглощения крови",
+	"mmorpg.stat.blood_per_10_strength": "§a[VAL1] §4❤ Крови§7 за 10 §b★ Силы§7",
+	"mmorpg.stat.blood_user": "§7Теперь вы используете §4❤ Кровь§7 вместо §9☯ Маны§7",
+	"mmorpg.stat.bonus_exp": "Бонус опыта",
+	"mmorpg.stat.bow_damage": "Урон от лука",
+	"mmorpg.stat.burn_chance": "Шанс на ожог",
+	"mmorpg.stat.burn_damage": "Урон от ожога",
+	"mmorpg.stat.burn_duration": "Продолжительность ожога",
+	"mmorpg.stat.burn_proc_chance": "Шанс на раскаленность",
+	"mmorpg.stat.burn_resistance": "Сопротивление ожогу",
+	"mmorpg.stat.burn_strength": "Сила ожога",
+	"mmorpg.stat.cast_speed": "Скорость каста",
+	"mmorpg.stat.cd_ticks": "Тики перезарядки",
+	"mmorpg.stat.cdr": "Снижение перезарядки",
+	"mmorpg.stat.chaining_cdr": "Снижение перезарядки цепляющих заклинаний",
+	"mmorpg.stat.chaining_spell_dmg": "Урон от цепляющих заклинаний",
+	"mmorpg.stat.chance_of_blind": "§7Ваши §5☀ Хаос§7 Атаки имеют §a[VAL1]§7% шанс на наложение Слепоты",
```

```suggestion
	"mmorpg.stat.chance_of_blind": "§7Ваши атаки §5☀ Хаосом§7 имеют §a[VAL1]§7% шанс на наложение Слепоты",
```
(это не единственный случай)

---

Исправлено